demo/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "@stoplight/elements": "^8.1.3",
-    "@stoplight/mosaic": "^1.53.1",
+    "@stoplight/mosaic": "1.53.3",
```

Comment:
^ is missing or removed intentionally?

Reply:
Updated the version. Please review once.